lib/deck.rb

```diff
@@ -0,0 +1,19 @@
+class Deck
+    attr_reader :cards, :category
```

Comment:
You have an `attr_reader` for `category` but I do not see an instance variable called `category` anywhere in the class.